helm/mds/requirements.yaml

```diff
@@ -2,10 +2,10 @@
 dependencies:
   - name: redis
     version: 10.5.7
-    repository: alias:stable
+    repository: alias:bitnami
```

Comment:
these changes are already in `develop` so i don't feel bad about having them here